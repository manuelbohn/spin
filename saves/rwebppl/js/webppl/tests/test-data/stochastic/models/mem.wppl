var f = mem(function(name) {
  return flip();
});

var model = function() {
  assert.ok(f('x') === f('x'));
  return f('x') + f('y');
};
