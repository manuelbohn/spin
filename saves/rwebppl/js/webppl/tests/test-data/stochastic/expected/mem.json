{
  "hist": {
    "0": 0.25,
    "1": 0.5,
    "2": 0.25
  }
}
